src/components/Faq/styles.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+// import { bold } from 'ansi-colors';
+// why cant it be style.title? shouldnt it create a tag <title> and I shiuld be able to style it
```

Comment:
It looks like its because the title tag is a tag that is only used in the head of the document, and isn't a tag you can style in the dom.
Check out the documentation here: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title